src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -408,7 +409,7 @@ public static void Copy(IntPtr source, IntPtr[] destination, int startIndex, int
         //====================================================================
         public static byte ReadByte([MarshalAs(UnmanagedType.AsAny), In] Object ptr, int ofs)
         {
-            throw new PlatformNotSupportedException(); // https://github.com/dotnet/coreclr/issues/10442
+            return ReadValueSlow(ptr, ofs, (IntPtr nativeHome, int offset) => { return Marshal.ReadByte(nativeHome, offset); });
```

Comment:
Thanks. I agree this looks simpler but given that Marshal.ReadByte has different overloads that takes object/IntPtr I'd like to make this a bit more explicit.